### ISSUE TITLE
Incomplete rule is parsed as property

### ISSUE THREAD

**Issue by ramya-rao-a** [emmetio/css-parser#3]
In the below example, there are 2 nodes in the parsed model. 
First is of type `rule` which is right
Second is of type `property` with `name` as `nav` which is clearly wrong.

```
.foo {
    margin: 1px;
}

nav
```

I am assuming that there cannot be properties outside of a rule.

**Comment by sergeche** [comment]
It’s possible to have variables like `$foo: bar;` or `@import url()` which resolved as properties

**Comment by ramya-rao-a** [comment]
But they are not really properties are they? Can we have some other node type for them? When I think "property" in the context of css, I think of css property value pairs inside rules

![image](https://user-images.githubusercontent.com/16890566/35654752-50ecbdf8-06a3-11e8-864e-a4abd26255a2.png)

**Comment by sergeche** [comment]
Yes, these are not exactly properties. Maybe you can check such faux properties as `node.type === 'property' && !node.parent`?

**Comment by ramya-rao-a** [comment]
I ended up doing something like that

`node.type === 'property' && (node.parent.type == 'rule' || node.parent.type == 'at-rule')`
